test/race_cond_test.go

```diff
@@ -0,0 +1,227 @@
+package main
+
+// Race Condition Tests
+// This file contains code necessary to test for racing conditions in DBSWriter APIs
+
```

Comment:
please provide reference here to github issue about racing condition and explain what this test suppose to do, e.g. how many servers we need, how many blocks we must submit to trigger racing condition, etc.